modules/cli/src/main/java/org/apache/ignite/cli/deprecated/ui/ProgressBar.java

```diff
@@ -107,7 +107,7 @@ private String render() {
 
         var completedPart = ((double) curr / (double) max);
 
-        // Space reserved for '||Done!'
+        // Space reserved for '||Done'
         var reservedSpace = 7;
```

Comment:
Should the resevred space be 6 now that the exclamation mark is removed?

Reply:
I set it to 6 first, and it broke ProgressBarTest. This number makes no differense on what the user sees.